datastore/load_test.go

```diff
@@ -602,6 +602,62 @@ func TestTimezone(t *testing.T) {
 	}
 }
 
+func TestLoadArrayIndex(t *testing.T) {
```

Comment:
This fails intermittently with error:

```
=== RUN   TestLoadArrayIndex
    load_test.go:657: NoIndex should be correct: compare:
        got:  &datastore.Entity{Key:(*datastore.Key)(0xc0005457c0), Properties:[]datastore.Property{datastore.Property{Name:"non-indexed", Value:[]interface {}{"3", "4"}, NoIndex:true}, datastore.Property{Name:"indexed", Value:[]interface {}{"1", "2"}, NoIndex:false}}}
        want: &datastore.Entity{Key:(*datastore.Key)(0xc0004b1b80), Properties:[]datastore.Property{datastore.Property{Name:"indexed", Value:[]interface {}{"1", "2"}, NoIndex:false}, datastore.Property{Name:"non-indexed", Value:[]interface {}{"3", "4"}, NoIndex:true}}}
--- FAIL: TestLoadArrayIndex (0.00s)
```

Please see suggested changes.

Reply:
thanks for the suggestion.